pkg/util/error.go

```diff
@@ -12,7 +12,14 @@ import (
 	"k8s.io/client-go/discovery"
 )
 
-var ErrClientUnauthorized = errors.New("ServerSideApply is unauthorized")
+const (
+	MsgTLSCertificateExpired = "expired certificate" // stdlib: crypto/tls/alert.go
```

Comment:
Just wondering if this could also be a private member of the ssa.go? Do we use it somewhere outside?